### DISCUSSION TITLE
where to ask questions regarding a specific homebrew package?

### DISCUSSION THREAD

**Discussion by AntonioSun** [Homebrew/discussions#4992] [Getting started] [answered]
### Output of `brew config`

```shell
skip
```

### Output of `brew doctor`

```shell
skip
```

### Description of issue

where to ask questions regarding a specific homebrew package?

**Comment by SMillerDev** [comment]
At the developers of that software, unless you are wondering about the packaging method. In that case you can ask here.

**Reply by AntonioSun** [reply]
Oh, OK. so not here. 

Then, given a name of homebrew package, how do I know the name of the developers / package builder of that software (and/or the best method for contacting them) pls? (like in Debian, there are package home and maintainer two meta fields)

**Reply by SMillerDev** [reply]
`brew home <package name>` will open the homepage

**Reply by AntonioSun** [reply]
Hmm... brew home <package name> opens the homepage _**of the package**_, but the mac package _**building**_ repo might be somewhere else, not mentioned in the homepage, or hard to find even if it does. Like trying to find the mac package building repo for `imagemagick` (for #4993) using `brew home imagemagick` turned out to be very challenging.

**Reply by SMillerDev** [reply]
Homebrew builds the software
Imagick develops the software

If you have questions about imagick: ask the imagick devs
If you have questions about how imagick was packaged for your brew install: ask here

**Reply by p-linnane** [reply, marked as answer]
@AntonioSun I think you are trying to ask who specifically maintains `imagemagick`. Homebrew does not operate that way. There are no dedicated maintainers for packages. Packages are updated by whoever opens a pull request.

If you are looking for the package definition, you can run `brew info imagemagick`. It will point you to https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/i/imagemagick.rb.

**Reply by AntonioSun** [reply]
Thanks @p-linnane, that's _**exactly**_ what I was looking for -- where to report the packaging problem. BINGO!
